src/Components/Light/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import "./style.css"
+import bulbOn from "./images/light-on.svg";
+import bulbOff from "./images/light-off.svg";
+
+
+const Light = ({ name, state }) => {
+    const [stateState, setState] = useState(state)
+    return (
+        <div className="light" onClick={() => { (stateState == "on") ? setState("off") : setState("on") }} >
+            <div className="light__icon">
+                <img src={(stateState == "on") ? bulbOn : bulbOff} ></img>
```

Comment:
chceš obecně radši používat 3 === 
https://stackoverflow.com/questions/16633947/in-javascript-is-it-better-to-use-two-equal-signs-or-three-when-comparing-varia